definitions/checks/foreman_facts_names.rb

```diff
@@ -0,0 +1,25 @@
+class Checks::ForemanFactsNames < ForemanMaintain::Check
+  metadata do
+    label :foreman_fact_names
+    tags :default
```

Comment:
This check uses a feature `:foreman_database`, could you add confine block?
So that this check will be available only when this feature exists.


```
confine do
  feature(:foreman_database)
end
```

Reply:
Sure had no idea.